src/utils/network/session-core.js

```diff
@@ -293,3 +305,72 @@ export const getBlobFromSolid = async (session, fileUrl) => {
   const fileBlob = await getFile(fileUrl, { fetch: session.fetch });
   return URL.createObjectURL(fileBlob);
 };
+
+/**
+ * A function fetches the user's profile information from their webId's profile
+ * card
+ *
+ * @function fetchProfileInfo
+ * @param {Session} session - Solid's Session Object {@link Session}
+ * @returns {Promise<object>} Object - The object containing the information related
+ * to the person on their profile card, the profile dataset, and the profile Thing
+ */
+export const fetchProfileInfo = async (session) => {
+  const profileDataset = await getWebIdDataset(session.info.webId);
+  const profileThing = getThing(profileDataset, session.info.webId);
+
+  const profileName = getStringNoLocale(profileThing, RDF_PREDICATES.profileName);
+  const memberOf = getStringNoLocale(profileThing, RDF_PREDICATES.memberOf);
```

Comment:
According to schema.org, the object of a `memberOf` predicate isn't a string. It's an Organization (which is a child of thing), so this schema is incorrect.

https://schema.org/Organization

Reply:
Switching this back to Organization.